src/backend/src/transformers/description-bullets.transformer.ts

```diff
@@ -0,0 +1,13 @@
+import { Prisma } from "@prisma/client";
+import { descBulletArgs } from "../prisma-query-args/description-bullets.query-args";
+
+
+export const descBulletTransformer = (descBullet: Prisma.Description_BulletGetPayload<typeof descBulletArgs>) => {
```

Comment:
should export as default like risks

also lets rename it to `descriptionBulletTransformer`